pmd-apex/src/test/java/net/sourceforge/pmd/lang/apex/rule/ApexXPathRuleTest.java

```diff
@@ -39,10 +39,15 @@ public void testFileNameInXpath() {
 
     @Test
     public void testBooleanExpressions() {
-        Report report = apex.executeRuleOnResource(makeXPath("//BooleanExpression[@Operator='&&']"),
+        Report report = apex.executeRuleOnResource(makeXPath("//BooleanExpression[@Op='&&']"),
                                                    "BooleanExpressions.cls");
         assertSize(report, 1);
     }
 
-
+    @Test
+    public void testOldOperatorProperty() {
+        Report report = apex.executeRuleOnResource(makeXPath("//BooleanExpression[@Operator='&&']"),
+                                                   "BooleanExpressions.cls");
+        assertSize(report, 0);  // retired
```

Comment:
The deprecated `getOperator` method was fully removed in this branch. The method returned a Jorje type and would block the above-mentioned goal to compile without the Jorje JAR... or faking some of its classes.

Since I updated some xpath tests, I added an old variant to investigate and capture the xpath behavior after removal.

This is another case where the merge back into pmd6 or 7 should preserve the deprecated method. (And it would also need a summit-to-jorje conversion to behave correctly.)